compiler/src/iree/compiler/Codegen/Common/IREEExpandStridedMetadata.cpp

```diff
@@ -45,6 +45,27 @@ static AffineMap getMulMap(MLIRContext *context) {
   return AffineMap::get(0, 2, s0 * s1);
 }
 
+/// Walks up the def-use chain to find the HAL interface binding subspan
+/// that a memref value derives from, passing through transparent wrapper ops.
+static IREE::HAL::InterfaceBindingSubspanOp
```

Comment:
Nit: Could you make this `std::optional<IREE::HAL::InterfaceBindingSubspanOp>` ?